src/printer.js

```diff
@@ -144,7 +144,8 @@ function genericPrint(path, options, print) {
       ]);
 
     case 'ExpressionStatement':
-      const addSemicolon = path.getParentNode().type !== 'ForStatement'
+      // eslint-disable-next-line no-case-declarations
```

Comment:
This rule’s [documentation](https://eslint.org/docs/rules/no-case-declarations) explains why this style is bad practice and suggests wrapping each case in a block.

Reply:
Oh, of course, wrapping it in a block is the right solution :expressionless: I completely forgot you could do that. Thanks!

Reply:
Done. I didn't do it in all cases because we have several open PRs and I don't want to cause a lot of merge conflicts. I think just doing it as it's needed will be fine.